src/daemon/protocol.h

```diff
@@ -69,12 +69,13 @@ class t_protocol final
 
   ~t_protocol()
   {
-    LOG_PRINT_L0("Deinitializing cryptonote_protocol...");
+    LOG_PRINT_L0("Stopping cryptonote protocol...");
     try {
       m_protocol.deinit();
       m_protocol.set_p2p_endpoint(nullptr);
+      LOG_PRINT_L0("Cryptonote protocol stopped successfully.");
```

Comment:
Could somebody maybe point me to the correct location in code following the destructor?